pkgs/by-name/sw/switchaudio-osx/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  xcodebuild,
+  xcbuildHook,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "switchaudio-osx";
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
+    owner = "deweller";
+    repo = "switchaudio-osx";
+    tag = version;
+    sha256 = "sha256-AZJn5kHK/al94ONfIHcG+W0jyMfgdJkIngN+PVj+I44=";
+  };
+
+  buildInputs = [ xcodebuild ];
+
+  nativeBuildInputs = [ xcbuildHook ];
+
+  postPatch = ''
```

Comment:
This kind of a patch would probably fit better in a local git diff file added to `patches` with a comment linking to the upstream PR